core/src/backend/ui.rs

```diff
@@ -94,6 +94,9 @@ pub trait UiBackend: Downcast {
     // Only used on web.
     fn open_virtual_keyboard(&self);
 
+    // Only used on web.
+    fn close_virtual_keyboard(&self);
```

Comment:
It _may_ also end up getting used in the native Android app, eventually, maybe...

Reply:
You're right, it might also be useful for desktop when we start supporting touch, so I've removed the comments :)